lib/colors.js

```diff
@@ -16,9 +16,10 @@ const plain = {
 }
 
 const { createColors } = require('colorette')
-const { white, bgRed, red, yellow, green, blue, gray, cyan } = createColors({ useColor: true })
+const availableColors = createColors({ useColor: true })
+const { white, bgRed, red, yellow, green, blue, gray, cyan } = availableColors
 
-const colored = {
+let colored = {
```

Comment:
Unfortunately this change is not ok. We should not be changing a singleton value in this way.

Reply:
You are totally right, I will adapt to avoid the singleton mutation.